source/_integrations/reolink.markdown

```diff
@@ -173,6 +174,8 @@ Depending on the supported features of the camera, switch entities are added for
 - FTP upload
 - HDR*
 
+When **Infra red lights in night mode** entity is OFF always OFF, when ON IR LEDs will be on when the camera is in night vision mode, see the **Day night mode** select entity.
```

Comment:
```suggestion
When the **Infrared lights in night mode** entity is set to OFF, the infrared LEDs are always OFF. When the **Infrared lights in night mode** entity is set to ON, the infrared LEDs will be on when the camera is in night vision mode. For more information, see the **Day night mode** select entity.
```
hm, not sure I understand this. Please correct me if I got this wrong.

Reply:
No you got it.